CMakeLists.txt

```diff
@@ -745,11 +750,31 @@ if(HPX_WITH_AGAS_DUMP_REFCNT_ENTRIES)
   hpx_add_config_define(HPX_HAVE_AGAS_DUMP_REFCNT_ENTRIES)
 endif()
 
+hpx_option(HPX_WITH_DISTRIBUTED_RUNTIME BOOL
+  "Enable the distributed runtime (default: ON). Turning off the distributed runtime completely disallows the creation and use of components and actions. Turning this option off is experimental!"
+  ON CATEGORY "Build Targets" ADVANCED)
```

Comment:
Would the checks above properly pick up this option?

Reply:
Excellent catch! I've fixed that now.